packages/dd-agent/packaging

```diff
@@ -101,3 +101,13 @@ pushd ${BOSH_INSTALL_TARGET}/embedded/lib/pkgconfig/
     fi
   done
 popd
+
+
+echo "Fixing python prefixes ..."
+sitecustomize_script="import site
```

Comment:
Could we echo line by line into the file instead? 
This way we could have this inside the `pushd`.

Reply:
doing that